internal/configuration/registry/usecase.go

```diff
@@ -58,6 +59,11 @@ type UseCaseRegistry struct {
 	availableLiquidityUseCase     *liquidity_provider.GetAvailableLiquidityUseCase
 	updatePeginDepositUseCase     *watcher.UpdatePeginDepositUseCase
 	getServerInfoUseCase          *liquidity_provider.ServerInfoUseCase
+	trustedAccountRepository      entities_lp.TrustedAccountRepository
```

Comment:
this is the use case registry, this shouldn't be here, there is a database registry